cla_backend/apps/reports/sql/CaseDemographicsReport.sql

```diff
@@ -0,0 +1,151 @@
+        JOIN call_centre_operator as op on u.id = op.user_id
+      WHERE
+        e.code != 'CASE_CREATED'
+
+  ),
+    operator_first_action as (
+      SELECT e.*
+        ,row_number() over (PARTITION BY e.case_id order by e.id asc) as rn
+      FROM
+        cla_eventlog_log as e
+        JOIN auth_user as u on e.created_by_id = u.id
+        JOIN call_centre_operator as op on u.id = op.user_id
+      WHERE
+        e.code != 'CASE_CREATED'
+        and e.level >= 29
+  ),
+    provider_first_view as (
+      SELECT
+        e.*
+        ,row_number() over (PARTITION BY e.case_id order by e.id asc) as rn
+      FROM
+        cla_eventlog_log AS e
+        JOIN auth_user AS u ON e.created_by_id = u.id
+        JOIN cla_provider_staff AS staff ON u.id = staff.user_id
+      WHERE
+        e.code != 'CASE_CREATED'
+
+  ), provider_first_assign as (
+    SELECT
+      e.*
+      ,row_number() over (PARTITION BY e.case_id order by e.id asc) as rn
+    FROM
+      cla_eventlog_log AS e
+    WHERE
+      e.code  in ('REFSP', 'MANALC', 'MANREF', 'SPOR')
+
+)
+select
+  c.laa_reference as "LAA_Reference"
+  ,md5(lower(regexp_replace((pd.full_name||pd.postcode)::text, '\s', '', 'ig'))) as "Hash_ID"
+  ,c.reference as "Case_ID"
+  ,COALESCE(assigned_provider.name, trim((log.context->'provider')::text, '"'), provider.name) as "Provider_ID" -- need to convert to LAA provider ID
+  ,category.code as "Category_Name"
+  ,c.created as "Date_Case_Created"
+  ,mt1.code as "Matter_Type_1"
+  ,mt2.code as "Matter_Type_2"
+  ,CASE diagnosis.state
+      when 'INSCOPE' then 'PASS'
+      when 'OUTOFSCOPE' then 'FAIL'
+      else 'UNKNOWN'
+  END as "Scope_Status"
+  ,CASE ec.state
+      when 'yes' then 'PASS'
+      when 'no' then 'FAIL'
+      else 'UNKNOWN'
+  END as "Eligibility_Status"
+  ,COALESCE(adapt.bsl_webcam, false)::bool as "Adjustments_BSL"
+  ,CASE upper(COALESCE(adapt.language, ''))
+      WHEN upper('English') THEN false
+      WHEN upper('Welsh') THEN false
+      WHEN '' THEN false
+      ELSE true
+  END as "Adjustments_LLI"
+  ,COALESCE(adapt.minicom, false)::bool as "Adjustments_MIN"
+  ,COALESCE(adapt.text_relay, false)::bool as "Adjustments_TYP"
+  ,COALESCE(adapt.callback_preference, false)::bool as "Adjustments_CallbackPreferred"
+  ,COALESCE(adapt.skype_webcam, false)::bool as "Adjustments_Skype"
+  ,CASE
+      WHEN EXTRACT(YEAR from age(now(), pd.date_of_birth)) <= 24 THEN 'A'
+      WHEN EXTRACT(YEAR from age(now(), pd.date_of_birth)) <= 29 THEN 'B'
+      WHEN EXTRACT(YEAR from age(now(), pd.date_of_birth)) <= 39 THEN 'C'
+      WHEN EXTRACT(YEAR from age(now(), pd.date_of_birth)) <= 49 THEN 'D'
+      WHEN EXTRACT(YEAR from age(now(), pd.date_of_birth)) <= 59 THEN 'E'
+      WHEN EXTRACT(YEAR from age(now(), pd.date_of_birth)) <= 79 THEN 'F'
+      WHEN EXTRACT(YEAR from age(now(), pd.date_of_birth)) >= 80 THEN 'G'
+      WHEN pd.date_of_birth IS NULL THEN 'U'
+      END as "Age(Range)"
```

Comment:
This matches [LGA 2685](https://dsdmoj.atlassian.net/jira/software/c/projects/LGA/boards/226?selectedIssue=LGA-2685), however, should the updated age ranges be reflected in the `MIExtractByOutcome` and `MIExtractByOutcomeExtended` reports? 
The ticket has ambiguously worded this change, and I am unsure how this should be interpreted.

Reply:
As its quite hard to find out who are the users of some of the original reports we will not be amending them, however the Extended one is recently created - going to see if that needs amending or just removing if it is succeeded by the new one - will amend or renew in a new pull request.

Reply:
It has been agreed with the user that we will not be amending any other reports.